### PR TITLE
Remove waitlist

### PR DESCRIPTION
**Pull request by eyalb181**

### CODE REVIEW

File: mirrord/cli/src/config.rs [hunk 1]
```diff
@@ -4,7 +4,6 @@ use std::path::PathBuf;
 
 use clap::{ArgGroup, Args, Parser, Subcommand, ValueEnum};
 use clap_complete::Shell;
-use email_address::EmailAddress;
```
Comment (t4lz) [inline mirrord/cli/src/config.rs:7]: Please also remove the dependency from `Cargo.toml`.

File: mirrord/cli/src/connection.rs [hunk 2]
```diff
@@ -108,7 +108,7 @@ where
         // This is CLI Only because the extensions also implement this check with better messaging.
         progress.print( "When targeting multi-pod deployments, mirrord impersonates the first pod in the deployment.");
         progress.print("Support for multi-pod impersonation requires the mirrord operator, which is part of mirrord for Teams.");
-        progress.print("To try it out, join the waitlist with `mirrord waitlist <email address>`, or at this link: https://metalbear.co/#waitlist-form");
+        progress.print("You can get started with mirrord for Teams at this link: http://mirrord.dev/docs/teams/introduction/");
```
Comment (t4lz) [inline mirrord/cli/src/connection.rs:111]: Looks better, no?
```suggestion
        progress.print("You can get started with mirrord for Teams at this link: https://mirrord.dev/docs/teams/introduction/");
```

File: mirrord/cli/src/error.rs [hunk 3]
```diff
@@ -240,7 +237,7 @@ pub(crate) enum CliError {
     InternalProxyError(#[from] IntProxyError),
     #[error("Feature `{0}` requires a mirrord operator.")]
     #[diagnostic(help(
-        "The mirrord operator is part of mirrord for Teams. To try it out, join the waitlist at this link: https://metalbear.co/#waitlist-form.{GENERAL_HELP}"
+        "The mirrord operator is part of mirrord for Teams. You can get started with mirrord for Teams at this link: http://mirrord.dev/docs/teams/introduction/"
```
Comment (t4lz) [inline mirrord/cli/src/error.rs:240]:
```suggestion
        "The mirrord operator is part of mirrord for Teams. You can get started with mirrord for Teams at this link: https://mirrord.dev/docs/teams/introduction/"
```

File: mirrord/cli/src/config.rs (hunk 1 above)
Comment (eyalb181) [reply mirrord/cli/src/config.rs:7]: Done

**Review by t4lz — APPROVED** [review]

**Pull request merged**